src/Logging.W3C/src/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,40 @@
+﻿Microsoft.Extensions.Logging.W3C.W3CLoggerFactoryExtensions
```

Comment:
This file wasn't copied from eng/PublicAPI.empty.txt

```suggestion
#nullable enable
﻿Microsoft.Extensions.Logging.W3C.W3CLoggerFactoryExtensions
```

Reply:
https://github.com/dotnet/aspnetcore/pull/33251/commits/fcf767132a91adb16682c7b8afc1f4a628194c23